lib/api/v1/serializers/keyword_detail_test.go

```diff
@@ -0,0 +1,37 @@
+package serializers_test
+
+import (
+	"github.com/markgravity/golang-ic/lib/api/v1/serializers"
+	"github.com/markgravity/golang-ic/test/fabricators"
+
+	_ "github.com/go-oauth2/oauth2/v4"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("KeywordDetailSerializer", func() {
+	Describe("#Data", func() {
+		Context("given a VALID params", func() {
```

Comment:
```suggestion
		Context("given VALID keyword", func() {
```

a... param**s** ... but in the end it's a keyword, right?